packages/office-ui-fabric-react/src/components/TextField/MaskedTextField/MaskedTextField.tsx

```diff
@@ -124,7 +124,7 @@ export class MaskedTextField extends BaseComponent<ITextFieldProps, IMaskedTextF
         onKeyDown={this._onKeyDown}
         onPaste={this._onPaste}
         value={this.state.displayValue}
-        ref={this._resolveRef('_textField')}
+        componentRef={this._resolveRef('_textField')}
```

Comment:
can we use React.createRef here?

Reply:
Since _resoleRef is deprecated, we could replace its usage throughout Fabric, but I'd rather do that in a separate PR.